gamedb.yaml

```diff
@@ -5287,3 +5287,39 @@
   id: Chicken
   banner: steam:1530300
   status: unsupported
+
+# Native version doesn't support steam achievements
+- name: '12 is Better Than 6'
+  platform: steam
+  id: 410110
+  compat_tool: proton-stable
+  status: playable
+  notes:
+   - Controller needs to be enabled in the game
+
+- name: 'The Metronomicon: Slay The Dance Floor'
+  id: 416790
+  platform: steam
+  compat_tool: GE-Proton8-15
```

Comment:
Is this specific version of GE Proton required? 8-25 is the latest that is available.

Reply:
Changed it to 8-25
When I tested it, only 8-21 was available which did not work.